test/mode/modeInsert.test.ts

```diff
@@ -215,6 +215,13 @@ suite('Mode Insert', () => {
     end: ['|abcd'],
   });
 
+  newTest({
+    title: 'Can <Esc> after entering insert mode from <ctrl+o>',
+    start: ['abc|'],
+    keysPressed: '<C-o>a<Esc>d',
```

Comment:
Don't you need to be in insert mode before hitting `<C-o>`? Also, use `endMode: ` to assert that we're in normal mode rather than hitting `d` and making sure it's not inserted.

Reply:
Fixed.

Reply:
Thanks for the feedback, didn't know about `endMode`. Fixed.